tests/steth/test_zap_steth.py

```diff
@@ -0,0 +1,29 @@
+from itertools import count
+from brownie import Wei, reverts
+import eth_abi
+from brownie.convert import to_bytes
+import random
+import brownie
+
+
+def test_zapper(strategy, token, zapper, chain, vault, whale, gov, interface, user):
```

Comment:
can add test for ERc20 after changes

Reply:
Maybe I am missing something, I got told we only care about the ETH zaps for now.

Reply:
See answer below